pkg/provision/mbus/api.go

```diff
@@ -128,3 +166,11 @@ func (d *Deployments) changesHandler(ctx context.Context, payload []byte) (inter
 	}
 	return data, nil
 }
+
+func (d *Deployments) listHandler(ctx context.Context, payload []byte) (interface{}, error) {
+	data, err := d.list(ctx, payload)
+	if err != nil {
+		return nil, err
+	}
+	return data, nil
```

Comment:
```suggestion
	return d.list(ctx, payload)
```

This should work, no ?